<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SaveEditor</class>
 <widget class="QWidget" name="SaveEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>164</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QComboBox, QLineEdit {
	background-color: #2b2b2b;
	border: none;
	border-bottom: 3px solid #6ABDDB;
	color: #a9b7c6;
	padding-left: 2px;
}

QComboBox::drop-down {
    border: 0;
}

QPushButton {
	background-color: #6ABDDB;
	color: #000000;
	border: none;
	padding: 10px;
	border-radius: 8px;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="saveEditorFrame">
     <property name="styleSheet">
      <string notr="true">#saveEditorFrame {
	background-color: #545454;
	/*border-radius: 8px;*/
	border-top: 3px solid #555555;
	border-bottom: 3px solid #282828;
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="saveLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">#saveLabel {
	color: #bbbbbb;
}</string>
        </property>
        <property name="text">
         <string>Save Request</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="projectComboBox">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>36</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="currentText">
         <string/>
        </property>
        <property name="frame">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="nameEdit">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>36</height>
         </size>
        </property>
        <property name="placeholderText">
         <string>Name</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="spacing">
         <number>15</number>
        </property>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="cancelSaveButton">
          <property name="styleSheet">
           <string notr="true">#cancelSaveButton {
	background-color: #795C06;
}</string>
          </property>
          <property name="text">
           <string>Cancel</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="confirmSaveButton">
          <property name="styleSheet">
           <string notr="true">QPushButton {
	background-color: #6ABDDB;
	color: #000000;
	border: none;
	padding: 10px;
	border-radius: 8px;
}</string>
          </property>
          <property name="text">
           <string>Confirm</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
